src/sql/workbench/common/workspaceActions.ts

```diff
@@ -27,6 +27,6 @@ export class OpenFileInFolderAction extends Action {
 	}
 
 	run() {
-		return this.openerService.open(URI.file(this.path));
+		return this.openerService.open(URI.file(this.path), { openExternal: true });
```

Comment:
I think previously (~a few months ago), CSV\XML\JSON would open in VS Code and XLSX would open externally.  Perhaps this was because vscode was the default file handler for these extensions?  In any case, I think we auto-open those files in any case, so the prompt was redundant for non-XLSX in all my testing.  

Do you know if this changes the CSV\XML\JSON behavior compared to a few months ago?  It likes like it will restore the XLSX behavior.

Reply:
this will restore the behavior is was before a few months ago (when they introduced the openerservice), basically it always tried to open it externally and let the os decide who to open it with.